casepro/backend/tests/test_junebug.py

```diff
@@ -693,14 +693,14 @@ def hub_outgoing_callback(request):
                 'inbound_channel_id': ''})
             headers = {'Content-Type': "application/json"}
             resp = {
-                'status': 201,
+                'status': 200,
                 'code': "created",
                 'description': "message submitted",
                 'result': {
                     'id': "message-uuid-1234",
                 },
             }
-            return (201, headers, json.dumps(resp))
+            return (200, headers, json.dumps(resp))
```

Comment:
Is the `200` -> `201` change to better represent what is actually being being returned by the API? Because `201` is a better descriptor for a creation endpoint than `200`

Reply:
Yeah
a) 200 is what the api endpoint is actually returning
b) that endpoint doesn't create anything. it just re-formats the data and then sends it to Jembi

Reply:
:+1: from me then :)